pom.xml

```diff
@@ -918,6 +918,9 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-surefire-plugin</artifactId>
           <version>${maven-surefire-plugin.version}</version>
+          <configuration>
+            <excludedGroups>flaky</excludedGroups>
+          </configuration>
```

Comment:
This change isn't consistent with the PR description "This adjustment ensures that all tests, including those marked as flaky, are executed as intended unless explicitly excluded".
Wouldn't this exclude the flaky group by default?

Reply:
@lhotari I would like to excluded flaky tests in maven command line, but can be easily run in IDE.

Reply:
Is this the original meaning of the "flaky" test group in the project? If so, please document this in the FlakyTest annotation. The PR description should also clearly express this that in CI, the tests aren't run at all.